regolith/tools.py

```diff
@@ -1287,4 +1289,44 @@ def get_id_from_name(coll, name):
                              case_sensitive=False)
     if person:
         return person["_id"]
-    else: return None
\ No newline at end of file
+    else: return None
+
+
+def get_grant_amount(grant, people, begin_date=None, end_date=None):
+    grant_amounts = []
+    if not grant.get('budget'):
+        return "This grant has no specified budget"
```

Comment:
can you print the id of the offending grant maybe?

Reply:
done